Prace_domowe/Praca domowa2/Grupa1/Sebastian_Deregowski/PD2_Sebastian_Deregowski.ipynb

```diff
@@ -0,0 +1,1544 @@
+{
```

Comment:
<p>lepiej policzyć pierwiastek z błędu średniokwadratowego - to w rzeczywistości jest RMSE</p><p> </p><p>Błąd należałoby wyliczyć tylko dla wartości imputowanych, bo gdy wyliczamy dla całej kolumny wynik jest zakłamany, ponieważ we wzorze jest wyliczana średnia, przez co liczność ma znaczenie.</p><p> </p><p><u>Prośba o zmodyfikowanie sposobu wyliczania błędu.</u></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>